app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -112,6 +125,33 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
         AppReviewUtils.showAppReview(this)
     }
 
+    private fun recordMultiTabExposure() {
+        val multiTabsExperiment = FocusNimbus.features.tabs
+        if (multiTabsExperiment.value().isMultiTab) {
+            multiTabsExperiment.recordExposure()
+        }
```

Comment:
```suggestion
        FocusNimbus.features.tabs.recordExposure()
```

You don't need to detect if the user is in an experiment; Nimbus already knows that.